cli/src/pcluster/configure/easyconfig.py

```diff
@@ -367,16 +368,22 @@ def prompt_os(self):
 
     def prompt_instance_types(self):
         """Ask for master_instance_type and compute_instance_type (if necessary)."""
+        default_master_instance_type = self.cluster_section.get_param_value("master_instance_type")
+        if not default_master_instance_type:
+            default_master_instance_type = get_default_instance_type()
```

Comment:
This shouldn't be needed if `pcluster_config` is loaded with `auto_refresh=True` (default value)

Reply:
We adde auto_refresh=False as part of this patch: https://github.com/aws/aws-parallelcluster/pull/2248 
Let's double check if we can move back to True.

Reply:
After a closer experiment, we still can't do auto_refresh. The problem is not about instance type this time, but about additional_iam_policy. As the code [here](https://github.com/aws/aws-parallelcluster/pull/2248/files#diff-4a0da8aa43fd2a763f436760124df48544177a813b12e02bc0c8dc87e33d8e8fL16) shown, with auto_refresh, the unnecessary policy for AWSBatch will be generated.

Reply:
OK, let's talk about this :+1: